app/perfsymboltable.cpp

```diff
@@ -730,9 +732,16 @@ static PerfAddressCache::SymbolCache cacheSymbols(Dwfl_Module *module, quint64 e
     for (int i = 0; i < numSymbols; ++i) {
         GElf_Sym sym;
         GElf_Addr symAddr;
-        const auto symbol = dwfl_module_getsym_info(module, i, &sym, &symAddr, nullptr, nullptr, nullptr);
-        if (symbol)
-            cache.append({symAddr - elfStart, sym.st_size, symbol});
+        Elf *elfp;
```

Comment:
this isn't used, so can be left at nullptr

Reply:
It is used later - in commit 3414df5

Reply:
please move it then to that commit